modules/core/src/smithy4s/http/HttpRestSchema.scala

```diff
@@ -0,0 +1,72 @@
+/*
+ *  Copyright 2021-2022 Disney Streaming
+ *
+ *  Licensed under the Tomorrow Open Source Technology License, Version 1.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *     https://disneystreaming.github.io/TOST-1.0.txt
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package smithy4s
+package http
+
+import smithy4s.PartialData
+import smithy.api.HttpPayload
+import smithy4s.schema._
+import smithy4s.schema.SchemaPartition.NoMatch
+import smithy4s.schema.SchemaPartition.SplittingMatch
+import smithy4s.schema.SchemaPartition.TotalMatch
+
+/**
+ * This construct indicates how a schema is split between http metadata
+ * (ie headers, path parameters, query parameters, status code) and body.
+ *
+ * The schema is split between two different
```

Comment:
minor: I think this comment is incomplete

Reply:
Addressed in [2e4b134](https://github.com/disneystreaming/smithy4s/pull/878/commits/2e4b134162c28801936b782a99ed7a83dd23c28f)